src/components/Form/Form.tsx

```diff
@@ -0,0 +1,74 @@
+import { useState } from 'react';
+import { Todo, User } from '../../types';
+
+import { InputTitle } from './InputTitle';
+import { SelectUser } from './SelectUser';
+import { getNewId } from '../../utils/getNewId';
```

Comment:
The `getNewId` function should not be imported and used directly in the `Form` component. According to the task requirements, data preparation, including ID generation, should be handled in the `App` component and passed down as props. Consider refactoring this part to align with the requirements.